src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -355,6 +355,17 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		e.browserEvent.preventDefault();
 		e.browserEvent.stopPropagation();
 
+		if ('target' in e.anchor) {
+			const { className } = e.anchor.target;
+			// Don't show the context menu for file trees in the response
+			if (className.includes('codicon-tree-item-expanded') ||
```

Comment:
You can probably figure out whether this element is below whatever class marks the file tree that we know about? These might hit on some other random sub element at some point

Reply:
🤡 I tried `preventDefault` but forgot `stopPropagation`, that did the trick